src/test/scala/wdl4s/expression/ValueEvaluatorSpec.scala

```diff
@@ -25,6 +25,7 @@ class ValueEvaluatorSpec extends FlatSpec with Matchers {
       WdlString("c") -> WdlInteger(2)
     ))
     case "o" => WdlObject(Map("key1" -> WdlString("value1"), "key2" -> WdlInteger(9)))
+    case "myPair" => WdlPair(WdlInteger(3), WdlString("hello"))
```

Comment:
TOL: Based on your comment above about being able to nest expressions in parens, "e.g. (1 + 2) + 3", it'd be nice to have an example of that as well.